react_app/src/components/Game/CTFCategory.js

```diff
@@ -20,13 +20,15 @@ const useStyles = makeStyles(theme => ({
 export default function CTFCategory(props) {
   const classes = useStyles();
 
-  const categoryColor = props.category.background ? props.category.background : "lightgrey"
-  if(props.category.challenges.length <= 0){
-    return null
+  const categoryColor = props.category.background
+    ? props.category.background
+    : "lightgrey";
+  if (props.category.challenges.length <= 0) {
```

Comment:
This will never be less than 0, so convention would be;
```
  if (!props.category.challenges.length) {
```
(because `0` is falsey)